content/introduction/getting-started-with-sdks/index.mdx

```diff
@@ -0,0 +1,319 @@
+- Example use cases
+
+## Overview
+
+Our SDKs simplify making requests to the backend by handling
+authentication and providing convenient access to Sharetribe's APIs.
+They allow you to interact easily with
+
+- Marketplace API, for general interactions with marketplace data
+- Asset Delivery API, for delivering assets configured in Console
+- Integration API, for advanced operational use
+
+You can find more information on API usage and guidelines for choosing
+which to work with in
+[this article](https://www.sharetribe.com/docs/concepts/api-sdk/marketplace-api-integration-api/#when-to-use-the-marketplace-api).
+
+There are two SDK variants in which to access the APIs
+
+<br />
+<Table>
+  <thead>
+    <Table.Tr>
+      <Table.Th />
+      <Table.Th>Sharetribe SDK</Table.Th>
+      <Table.Th>Sharetribe Integration SDK</Table.Th>
+    </Table.Tr>
+  </thead>
+  <tbody>
+    <Table.Tr>
+      <Table.Td>Intended Use</Table.Td>
+      <Table.Td>
+        Most common API requests with Marketplace API or Asset Delivery
+        API
+      </Table.Td>
+      <Table.Td>Operator-level requests with Integration API</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>Required Parameters</Table.Td>
+      <Table.Td>Client ID</Table.Td>
+      <Table.Td>Client ID, Client secret</Table.Td>
+    </Table.Tr>
+    <Table.Tr>
+      <Table.Td>Secure context needed</Table.Td>
+      <Table.Td>No</Table.Td>
+      <Table.Td>**Yes**</Table.Td>
+    </Table.Tr>
+  </tbody>
+</Table>
+
+To try out the Sharetribe SDK, visit our
+[Github documentation](https://sharetribe.GitHub.io/flex-sdk-js/index.html)
+for instructions on testing it directly in your browser or via the
+[API Playground](https://sharetribe.GitHub.io/flex-sdk-js/try-it-in-the-playground.html).
+For a more detailed guide on using the SDK with code, keep following
+this tutorial.
+
+## Installation
+
+To get started with the SDK, you'll first need to install it with either
+[Yarn](https://yarnpkg.com/)
+
+```jsx
+yarn add sharetribe-flex-sdk
+```
+
+or [npm](https://www.npmjs.com/)
+
+```jsx
+npm install sharetribe-flex-sdk
+```
+
+<Callout type="info">
+  The above example will create an instance that allows you to access
+  Marketplace API and Asset Delivery API. In order to access Integration
+  API, run `yarn add sharetribe-flex-integration-sdk`
+</Callout>
+
+## SDK Initialization Overview
+
+### API applications
+
+To use the Marketplace API or Integration API, you will need a **client
+ID**.
+
+When you get access, you will be able to log into Sharetribe Console and
+check the client ID.<br /> Sharetribe Console: _Build > Advanced >
+Applications_
+
+In addition, Sharetribe Web Template uses transaction processes that
+include privileged transitions. This makes it possible to customize
+pricing on the Node server that's included in the template. The **client
+secret** is needed to make this secure call from the template's own
+server to Sharetribe API.
+
```

Comment:
My suggestion would not be to mention the template here as the primary example. Privileged transitions can be used regardless of template usage or not. Technically, the client secret is needed to fetch the Trusted user access token, which is needed to call certain endpoints, including the transition transaction endpoint when transitioning a privileged transition. These calls should be made from a secure context. You could explain that and then illustrate how that works in practice by using the template as an example?